Sources/FuzzilliCli/Profiles/V8Profile.swift

```diff
@@ -270,10 +275,24 @@ let v8Profile = Profile(
         return args
     },
 
+    processArgumentsReference: [
```

Comment:
Here and in the other two profiles, I think it'd be nice to split these into `commonArguments`, `optimizingArguments`, and `referenceArguments` or so, so that e.g. `--harmony` is only specified in one place.